include/aws/common/logger.h

```diff
@@ -0,0 +1,104 @@
+#ifndef AWS_COMMON_LOGGER_H
+#define AWS_COMMON_LOGGER_H
+/*
+* Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License").
+* You may not use this file except in compliance with the License.
+* A copy of the License is located at
+*
+*  http://aws.amazon.com/apache2.0
+*
+* or in the "license" file accompanying this file. This file is distributed
+* on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+* express or implied. See the License for the specific language governing
+* permissions and limitations under the License.
+*/
+#include <aws/common/common.h>
+#include <stdarg.h>
+
+enum aws_log_level
+{
+    AWS_LOG_LEVEL_OFF = 0,
+    AWS_LOG_LEVEL_FATAL = 1,
+    AWS_LOG_LEVEL_ERROR = 2,
+    AWS_LOG_LEVEL_WARN = 3,
+    AWS_LOG_LEVEL_INFO = 4,
+    AWS_LOG_LEVEL_DEBUG = 5,
+    AWS_LOG_LEVEL_TRACE = 6
+};
+
+static inline const char *aws_get_log_level_name(enum aws_log_level level) {
```

Comment:
Trivial, but I don't think `static` does anything here. This is a header file

Reply:
If static is not provided, the symbol will be exported (at least at the library level) and will clash with any other .c files which included this header. In addition, it'll be generated even if it's not used.

That said this should probably not be static or inline.